docs/process/release.md

```diff
@@ -138,26 +141,28 @@ After closing the tab, you have successfully logged in, it is time to publish.
 
 **It is a good idea to wait for CI to be green on the patch branch and tag before doing the release.**
 
-Check out the patch tag (e.g. `v6.7.8`), then run:
-```sh
-devkit-admin publish
+Check out the patch branch (e.g. `9.1.x`), then run:
+```bash
+yarn # Reload dependencies
+yarn admin publish
 ```
 
-Check out the minor tag (e.g. `v6.8.0-beta.0`), then run:
+If also publishing a prerelease, check out `master`, then run:
 ```bash
-devkit-admin publish --tag next
+yarn # Reload dependencies
+yarn admin publish --tag next
 ```
 
 ### Release Notes
 
-`devkit-admin changelog` takes `from` and `to` arguments which are any valid git
+`./bin/devkit-admin changelog` takes `from` and `to` arguments which are any valid git
```

Comment:
On line 147 and 153 we use `yarn admin`, but here it's different. Could you please make them consistent?

Reply:
The changelog output is typically copied, but running `yarn admin` also prints:

```
yarn run v1.22.0
$ node ./bin/devkit-admin
```

Which we don't want in the changelog, so we shouldn't use `yarn admin` here. I could switch everything back to `./bin/devkit-admin` for consistency if that's better.

Reply:
Perhaps we could silent the output with `yarn run -s admin`?

Reply:
Done. Wasn't aware of that flag, thanks for pointing it out!